src/styles/shop.css

```
@@ -10,43 +10,45 @@
+}
+
+.shop-product-card:hover {
+  background-color: var(--shop-product-card-hover);
+}
+
+.shop-merch-mark {
+  display: inline-block;
+  width: 0.82em;
+  height: 0.82em;
+  flex: none;
+  background-color: currentColor;
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)